server/src/test/java/com/objectcomputing/checkins/services/guild/member/GuildServicesImplTest.java

```diff
@@ -313,104 +316,80 @@ void testFindByFieldsNullParams() {
                 new GuildMember(UUID.randomUUID(), UUID.randomUUID(), false)
         );
 
-        when(guildMemberRepository.findAll()).thenReturn(guildMemberSet);
+        when(guildMemberRepository.search(null, null, null)).thenReturn(guildMemberSet);
 
         assertEquals(guildMemberSet, services.findByFields(null, null, null));
 
-        verify(guildMemberRepository, times(1)).findAll();
+        verify(guildMemberRepository, times(1)).search(null, null, null);
```

Comment:
Because we aren't building the query based on which methods get called these verifies aren't useful anymore. We know the bottom three never get called because they don't exist in the module we're testing and we know the first DOES get called because if it doesn't the assert would fail.

Reply:
Probably applicable to all the references to `verify` in the other test for `findByFields`.